src/seraphis_wallet/key_container.h

```diff
@@ -0,0 +1,149 @@
+#pragma once
+
+// local headers
+#include "crypto/chacha.h"
+#include "crypto/crypto.h"
+#include "crypto/hmac-keccak.h"
+#include "seraphis_mocks/jamtis_mock_keys.h"
```

Comment:
As a general design decision, we probably shouldn't be making production code depend upon mock code.

Reply:
This looks like a job for somebody to pick up: Write a class that is meant to be definit / permanent and replace this mock-only class, right? Maybe all that is needed is an exact one-to-one copy of that code, but in another file in another Seraphis-related folder.

Not sure whether this file crept into other, already existing parts of the Seraphis wallet, and would need replacement there somehow as well.